### ISSUE TITLE
Example in README not accessible (and more intimidating than useful)

### ISSUE THREAD

**Issue by khinsen** [dreamtools/dreamtools#63]
The README provides a few example invocations, which I expected to be suggestions for trying out `dreamtools`. However, they use a dataset that apparently is not openly available.

Below is the output I get from the first example, `dreamtools --challenge D8C1 --onweb`. Please consider the impression that a new user gets from this output.

 1. It starts by asking me to add a README.rst to D8C1. But I am not the author of D8C1, I just want to consult this entry!
 2. It tells me that some file is missing. Why should I care? I asked `dreamtools` to open a Web page for me. Why should this require a local file on my computer?
 3. "This is a SynapseClient built on top of Synapse class." Nice to meet you, my name is Konrad.
 4. "Trying to login automatically". Login to what? Why? Just to open a Web page?
 5. "Create a Synapse login on http://synapse.org". I have one, but you never asked me about this.
 6. "Create a file called .synapseConfig in your HOME directory". Shouldn't this be part of the installation/setup instructions?
 7. "Traceback (most recent call last):"  This starts to look very bad...
 8. "WARNING: This entity has access restrictions."  Then why is it proposed as an example ???
 9. "Please visit the web page for this entity (syn.onweb("syn1920412"))." Visiting the Web site is exactly what I asked `dreamtools` to do for me !!!!! And what I am supposed to make of "syn.onweb("syn1920412")" ??? Is that some magic incantation?

In summary, all of this looks like debug output for `dreamtools` developers, not like anything of much help to a user.

```
Missing README.rst. Please add one in D8C1
Missing README.rst. Please add one in D8C1
File /Users/hinsen/Library/Application Support/dreamtools/dream8/D8C1/experimental.zip not found. Downloading from Synapse.
Missing README.rst. Please add one in D8C1
This is a SynapseClient built on top of Synapse class.
Trying to login automatically.
Failed to login automatically.
Either the login is wrong or missing
You must have a synapse account
Create a Synapse login on http://synapse.org
Create a file called .synapseConfig in your HOME directory
Add this code in the file:

[authentication]
For now, let us try to login manually:
username: yourlogin
password: yourpassword

Synapse username:khinsen
Synapse password: *****
Welcome, khinsen!
Traceback (most recent call last):
  File "./bin/dreamtools", line 9, in <module>
    load_entry_point('dreamtools==1.1.1', 'console_scripts', 'dreamtools')()
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/dreamtools/core/scoring.py", line 114, in scoring
    class_inst.import_scoring_class()
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/dreamtools/core/challenge.py", line 164, in import_scoring_class
    class_inst = getattr(py_mod, self.alias)()
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/dreamtools/dream8/D8C1/scoring.py", line 66, in __init__
    self._init()
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/dreamtools/dream8/D8C1/scoring.py", line 70, in _init
    self._download_data('experimental.zip', 'syn1920412')
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/dreamtools/core/challenge.py", line 178, in _download_data
    d.download(synid)
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/dreamtools/core/downloader.py", line 57, in download
    self.client.get(synid, downloadLocation=self.directory)
  File "/Users/hinsen/Temp/dreamtools-test/lib/python2.7/site-packages/synapseclient/client.py", line 653, in get
    raise SynapseUnmetAccessRestrictions(warning_message)
synapseclient.exceptions.SynapseUnmetAccessRestrictions:
WARNING: This entity has access restrictions. Please visit the web page for this entity (syn.onweb("syn1920412")). Click the downward pointing arrow next to the file's name to review and fulfill its download requirement(s).
```

**Comment by cokelaer** [comment]
Dear Konrad, 

You're right that there is no need to download files just to open a webpage. I will update the code accordingly (in issue #66 ). As for the missing files (README), this will be fixed (in issue #64 ).

I did not realise that the synapse entry was not accessible. I will work on fixing the example and tests during the next days and come back to you. 

Thanks for your comments and patience. 
Thomas

**Comment by cokelaer** [comment]
Comments on the different issues:

1 - The missing README file has been added to the release of dreamtools
2 - the --onweb option is now more robust and do not need to download any files anymore. 
     The --info option does not need to access to the web anymore (synapse)
3 - The informative message about synpaseclient has been removed.
4,5,6 - The messages related to the synapse login should be less verbose and more informative.
7,8,9: in some challenges (e.g., D8C1 from the quick start/tutorial) require a user to accept the conditions of use within synapse. This is outside of dreamtools somehow but we need to be more user-friendly indeed. So, the synapse page will be open and a message tells the user what to do (accept the conditions of use/ accept the terms).

The new version should be easier to use. The documentation still needs to be updated.
